app/kumactl/cmd/config/config_control_planes_add_test.go

```diff
@@ -84,9 +90,9 @@ var _ = Describe("kumactl config control-planes add", func() {
 			// when
 			err := rootCmd.Execute()
 			// then
-			Expect(err).To(MatchError(`Control Plane with name "example" already exists`))
+			Expect(err).To(MatchError(`could not add the control plane: Control Plane with name "example" already exists`))
```

Comment:
I think, we should add a couple new test cases:
1) test case where request to Control Plane url times out
2) test case where request to Control Plane url returns non-JSON / JSON of some other app

Reply:
added both tests